packages/launchpad/src/layouts/HeaderBar.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import { HeaderBarFragmentDoc } from '../generated/graphql-test'
+import HeaderBar from './HeaderBar.vue'
+
+describe('<HeaderBar />', () => {
+  it('renders with functional browser menu when show-browsers prop is true', () => {
+    cy.viewport(1000, 750)
```

Comment:
Fun fact, there's a loosely documented API to set this within the it/describe so that the test doesn't "jump" between one viewport size and another...

I don't use it all the time, but it's there if you want it 😄 

```js
describe('my suite', { /* anything in cypress.json */ viewportWidth: 1000, viewportHeight: 750 }, () => {
  it('renders', () => {
    /* ... */
  })
})
```